responses/fr/HassGetWeather.yaml

```diff
@@ -3,22 +3,200 @@ responses:
+        {% endif %}
+
+      # Starting from here, heavily inspired by the Hungarian template, kudos to RViktor
+      wind: >
+        {% set wind_speed = state.attributes.get('wind_speed') %}
+        {% set wind_speed_unit = state.attributes.get('wind_speed_unit', '') %}
+        {% set wind_bearing = state.attributes.get('wind_bearing') %}
+        {% set wind_gust_speed = state.attributes.get('wind_gust_speed') %}
+
+        {% set wind_scale = '' %} {# Beaufort scale #}
+        {% if wind_speed and wind_speed_unit == 'km/h' %}
+          {% if wind_speed <= 1 %}
+            {% set wind_scale = 'Vent calme' %}
+          {% elif wind_speed <= 5 %}
+            {% set wind_scale = 'Très légère brise' %}
+          {% elif wind_speed <= 11 %}
+            {% set wind_scale = 'Légère brise' %}
+          {% elif wind_speed <= 19 %}
+            {% set wind_scale = 'Petite brise' %}
+          {% elif wind_speed <= 28 %}
+            {% set wind_scale = 'Jolie brise' %}
+          {% elif wind_speed <= 38 %}
+            {% set wind_scale = 'Bonne brise' %}
+          {% elif wind_speed <= 49 %}
+            {% set wind_scale = 'Forte brise' %}
+          {% elif wind_speed <= 61 %}
+            {% set wind_scale = 'Grosse brise' %}
+          {% elif wind_speed <= 74 %}
+            {% set wind_scale = 'Coup de vent' %}
+          {% elif wind_speed <= 88 %}
+            {% set wind_scale = 'Fort coup de vent' %}
+          {% elif wind_speed <= 102 %}
+            {% set wind_scale = 'Tempête' %}
+          {% elif wind_speed <= 117 %}
+            {% set wind_scale = 'Violente tempête' %}
+          {% else %}
+            {% set wind_scale = "vent d'ouragan" %}
+          {% endif %}
+        {% elif wind_speed %}
+          {# Other wind speed unit, no scale, just the speed #}
+          {% set wind_scale = 'Vent' %}
+        {% endif %}
+
+        {% set wind_bearing_text = '' %}
+        {% if wind_bearing %}
+          {% if wind_bearing < 45 %}
+            {% set wind_bearing_text = 'nord' %}
+          {% elif wind_bearing < 90 %}
+            {% set wind_bearing_text = 'nord-est' %}
+          {% elif wind_bearing < 135 %}
+            {% set wind_bearing_text = 'est' %}
+          {% elif wind_bearing < 180 %}
+            {% set wind_bearing_text = 'sud-est' %}
+          {% elif wind_bearing < 225 %}
+            {% set wind_bearing_text = 'sud' %}
+          {% elif wind_bearing < 270 %}
+            {% set wind_bearing_text = 'sud-ouest' %}
+          {% elif wind_bearing < 315 %}
+            {% set wind_bearing_text = 'ouest' %}
+          {% elif wind_bearing <= 360 %}
+            {% set wind_bearing_text = 'nord-ouest' %}
+          {% else %}
+            {% set wind_bearing_text = 'inconnue' %}
+          {% endif %}
+        {% endif %}
+
+        {% if wind_speed_unit == 'km/h' %}
+          {% set wind_speed_unit = 'kilomètres par heure' %}
+        {% endif %}
```

Comment:
The native wid speed unit on the weather entity can be
`m/s, km/h, mi/h, ft/s or kn`

I would say, let's support them all, especially for our french-speaking colleague leaving elsewhere

Reply:
In French, `kn` would give `nd` for `noeuds`, idk what is the best :)